src/shared/logger.ts

```diff
@@ -4,11 +4,11 @@ export class Logger {
   }
 
 
-  log(message, isError, container) {
+  log(message, isError, container?) {
```

Comment:
container variable is optional, which resolved an error where the compiler thought log() required 3 parameters while it was called with 2 parameters